File: Examples/VisualBasic/WorkingWithResourceAssignments/GetResourceAssignmentPercentWorkComplete.vb

Namespace WorkingWithResourceAssignments
    Public Class GetResourceAssignmentPercentWorkComplete
        Public Shared Sub Run()
            ' The path to the documents directory.
            Dim dataDir As String = RunExamples.GetDataDir(System.Reflection.MethodBase.GetCurrentMethod().DeclaringType.FullName)

            ' ExStart:GetResourceAssignmentPercentWorkComplete
            ' Create project instance
            Dim project1 As New Project(dataDir + "ResourceAssignmentPercentWorkComplete.mpp")

            ' Print assignment percent completion
            For Each ra As ResourceAssignment In project1.ResourceAssignments
                Console.WriteLine(ra.[Get](asn.PercentWorkComplete).ToString())
            Next
            ' ExEnd:GetResourceAssignmentPercentWorkComplete
        End Sub
    End Class
End Namespace

File: Examples/CSharp/WorkingWithResourceAssignments/GetResourceAssignmentPercentWorkComplete.cs
namespace Aspose.Tasks.Examples.CSharp.WorkingWithResourceAssignments
{
    using System;

    internal class GetResourceAssignmentPercentWorkComplete
    {
        public static void Run()
        {
            // The path to the documents directory.
            var dataDir = RunExamples.GetDataDir(System.Reflection.MethodBase.GetCurrentMethod().DeclaringType.FullName);

            //ExStart:GetResourceAssignmentPercentWorkComplete
            //ExFor: Asn.PercentWorkComplete
            //ExSummary: Shows how to read percent work complete of an assignment.
            // Create project instance
            var project = new Project(dataDir + "ResourceAssignmentPercentWorkComplete.mpp");

            // Print assignment percent completion
            foreach (var ra in project.ResourceAssignments)
            {
                Console.WriteLine(ra.Get(Asn.PercentWorkComplete).ToString()); 
            }
            //ExEnd:GetResourceAssignmentPercentWorkComplete
        }
    }
}
